nautilus_trader/adapters/bybit/schemas/market/ticker.py

```diff
@@ -0,0 +1,169 @@
+# -------------------------------------------------------------------------------------------------
+#  Copyright (C) 2015-2023 Nautech Systems Pty Ltd. All rights reserved.
+#  https://nautechsystems.io
+#
+#  Licensed under the GNU Lesser General Public License Version 3.0 (the "License");
+#  You may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at https://www.gnu.org/licenses/lgpl-3.0.en.html
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# -------------------------------------------------------------------------------------------------
+
+import msgspec
+
+from nautilus_trader.adapters.bybit.schemas.common import BybitListResult
+from nautilus_trader.core.data import Data
+
+
+class BybitTickerData(Data):
```

Comment:
If you make this a type of `Ticker`, then you don't need to define a custom data type, and data will be returned through the normal `Ticker` methods and channels (see Binance for a similar pattern).

Reply:
I don't see where we're initializing `ts_init` or `ts_event`?